src/Management/test/Endpoint.Test/SpringBootAdminClient/SpringBootAdminClientHostedServiceTest.cs

```diff
@@ -43,6 +43,7 @@ public async Task SpringBootAdminClient_RegistersAndDeletes()
 
         httpMessageHandler.VerifyNoOutstandingExpectation();
         Assert.Equal("1234567", SpringBootAdminClientHostedService.RegistrationResult?.Id);
+        SpringBootAdminClientHostedService.RegistrationResult = null;
```

Comment:
Can you wrap this in a try/finally block? It likely affects the outcome of other tests if this one fails.